Fritz.StreamTools/Views/Followers/PreviewGoal.cshtml

```diff
@@ -101,20 +154,30 @@
 
 			function onload() {
 
-				var elements = document.getElementsByTagName("input");
-				for (var el of elements) {
+				for (var i = 0; i < localStorage.length; i++) {
```

Comment:
I switched from looping through elements on the page to looping through localstorage so that we import and re-create the dynamic color pickers.

Reply:
This looks REALLY cool... I'll review it on stream on Tuesday.  Thank you VERY much for the contribution!